src/docs/tutorial-transaction-process/customize-pricing/index.md

```diff
@@ -0,0 +1,482 @@
+---
+title: Customize pricing
+slug: customize-pricing-tutorial
+updated: 2020-07-21
+category: tutorial-transaction-process
+ingress:
+  Learn how to customize pricing in your marketplace by adding an optional
+  cleaning fee on top of the regular nightly price of the accommodation.
+published: true
+---
+
+## Store cleaning fee into listing
+
+Pricing can be based on a lot of variables but one practical way to build it is
+to base it on information stored as extended data in listings. In this example,
+we are using listing's public data to store information about the cleaning fee.
+We start by doing some changes to **EditListingPricingPanel** in
+_EditListingWizard_.
+
+```shell
+└── src
+    └── components
+        └── EditListingPricingPanel
+            └── EditListingPricingPanel.js
+```
+
+Here we will only list the required changes. See the
+[Add extended data to listing entity tutorial](/tutorial-extended-data/add-extended-data/)
+for step-by-step instructions on how to add public data field for listing.
+
+### Save to public data
+
+In _EditListingPricingPanel_ we need to edit the `onSubmit` function to save the
+new public data field `cleaningFee`. Because we are using `FieldCurrencyInput`
+in this example as the input, the `cleaningFee` will be Money object when we get
+that from `values`. Money object can't be used directly as public data so we
+need to create a JSON object with keys `amount` and `currency` and use the JSON
+object in the API call.
```

Comment:
This is a bit personal reference, but these yellow code-block highlights look pretty ugly. I've started to reduce the amount of code-block highlights in favour of italics and bolded words. Could you check if this would look better:


```suggestion
In _EditListingPricingPanel_, we need to edit the _onSubmit_ function to save the
new public data field called **cleaningFee**. Because we are using _FieldCurrencyInput_ component
in this example as the input of choice, the _cleaningFee_ variable will be Money object when we get
that from the submitted _values_. Money object can't be used directly as public data so we
need to create a JSON object with keys **amount** and **currency** and use it in the underlying API call.
```

Reply:
I changed the higlights to mostly italics now. I'm not sure what would be the best logic to choose between bolding or italics. Do you have any "rules" for this?

Maybe a better solution would be to discuss with Janne about changing the highlight formatting so that it would look nicer. I think it has been a common convention to use the highlight for variable, component, and file names quite some time already (or at least I have been doing that) so it might be confusing that we suddenly change this.

Reply:
@OtterleyW The highlight colour could be changed, but still I think the way we (and especially me) have used it is bad. The plethora of code-blocks makes it difficult to read the text. I still use it, but I've tried to minimize its usage in new articles to code expressions (not single words) and only max ~one per paragraph.

The problem is that code-block is not made to highlighting text, but to separate code statements (with ifs, elses and other English words) from a text that is part of a readable sentence. For example, assistive text-to-speech services probably can't make any sense of paragraphs where we have used code-blocks that much.